### DISCUSSION TITLE
Implementing diffusion on a hexagonal grid

### DISCUSSION THREAD

**Discussion by bzm3r** [Leafwing-Studios/Emergence#10] [General]
A relevant resource to consult: [Coordinate system for hexagonal pixels](https://www.academia.edu/download/46987496/A_Coordinate_System_for_Hexagonal_Pixels20160703-23011-1qrep9g.pdf)

Abstract: 

> A coordinate system is described which provides a natural means for representing
hexagonally-organized pixels. The coordinate system has the unusual property that its basis
vectors are not orthogonal. Vector-space properties and operations are described in this
coordinate system, and shown to be straightforward computations. Some image processing
algorithms, including calculations of image gradients and *variable-conductance diffusion*,
are expressed and analyzed.

**Comment by alice-i-cecile** [comment]
Related actionable issues: #1, #2.
